resqpy/property/_collection_create_xml.py

```diff
@@ -108,6 +107,7 @@ def _create_xml_patch_node(collection, p_node, points, const_value, indexable_el
 def _create_xml_property_min_max(collection, property_array, const_value, discrete, add_min_max, p_node, min_value,
                                  max_value):
     if add_min_max:
+        assert property_array is not None or const_value is not None
```

Comment:
Should we allow a min_value and max_value to be passed through to here, without the need for property_array or const_value?  I think we can just scrap this assertion altogether.